### PR TITLE
Client data

### PR DESCRIPTION
**Pull request by lexarisa**

### CODE REVIEW

**Review by thaistcosta — APPROVED** [review]
Loved it little 🦞 ❤️ !

File: client/common/components/BoardSection.tsx [hunk 1]
```diff
@@ -27,31 +31,68 @@ const BoardSection: React.FC<BoardInterface> = (props) => {
...
 
-  const onClick = () => {
-    console.log('hi');
+  const handleShowInput = () => {
+    setShowInput(!showInput);
+    setShowButton(false);
   };
+
+  const handleCreateTask = async () => {
+    await createNewTask(taskTitle);
```
Comment (alexandra-plana) [inline client/common/components/BoardSection.tsx:40]: should we also pass in the other inputs?

**Review by alexandra-plana — APPROVED** [review]
looks so goooooood!!!!! ✨

File: client/common/components/BoardSection.tsx (hunk 1 above)
Comment (lexarisa) [reply client/common/components/BoardSection.tsx:40]: yess :') trying for now

**Pull request merged**